ipn/ipnlocal/local.go

```diff
@@ -3119,6 +3129,10 @@ func (b *LocalBackend) TCPHandlerForDst(src, dst netip.AddrPort) (handler func(c
 		opts = append(opts, ptr.To(tcpip.KeepaliveIdleOption(72*time.Hour)))
 		return b.handleSSHConn, opts
 	}
+	// TODO(will,sonia): allow customizing web client port ?
```

Comment:
could put in a var for now? then also use from ipnlocal/web

Reply:
this PR removes the use of the port in ipnlocal/web, so this becomes the only place its used.  So pulling out into a var now doesn't do much.  I figure we'll do that when we make it configurable though.